src/components/kpi/kpi.less

```diff
@@ -39,7 +39,7 @@
   }
 
   &.editing {
-    min-height: @kpi-max-height + 60px;
+    min-height: @kpi-max-height + 80px;
```

Comment:
A good behaviour would be to add a comment to know what are those 80px, or in a local variable (avoid magic numbers)